lib/tasks/import_prebakedstats.rake

```diff
@@ -0,0 +1,73 @@
+require 'csv'
+
+namespace :import do
+
+  desc "import CSV data into database"
+  task :prebakedstats, [:csv_file] => [:environment] do
+    habitats = Habitat.all
+    geo_entity_types = ["countries", "regions"].freeze
+
+    # import habitat data CSVs for each entity type
+    geo_entity_types.each do |geo_entity_type|
+      habitats.each do |habitat|
+        csv_file = "#{geo_entity_type}/#{habitat.name}.csv"
+        import_new_csv_file(geo_entity_type, habitat.name, csv_file)
+      end
+    end
+  end
+
+  def import_new_csv_file(geo_entity_type, habitat, csv_file)
+    filename = "#{Rails.root}/lib/data/#{csv_file}"
+    CSV.foreach(filename, headers: true, encoding: "utf-8") do |row|
+      if row.select {|k,v| /baseline/i =~ k}.any?
+        parse_change_stat row, geo_entity_type, habitat
+      else
+        parse_geo_entity_stat row, geo_entity_type, habitat
+      end
+    end
+  end
+
+  def parse_change_stat csv_row, geo_entity_type, habitat
+    geo_entity = fetch_geo_entity(csv_row["name"] | csv_row["Region"], csv_row["iso3"])
```

Comment:
I think you meant `||`.
Single `|` is a bitwise operator which is not achieving the intended behaviour here I think.
For example, if one between `csv_row["name"]` and `csv_row["Region"]` is populated you will get `true`, `false` otherwise, but the first param for `fetch_geo_entity` is expecting a name.

Reply:
Thanks!

Reply:
@Levia I changed to `||` but I don't get the correct statistics post import. I will investigate further why this is happening.